src/fluent-plugin-mdsd/gemspec-template

```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+    s.name = "fluent-plugin-mdsd"
+    s.version = "GEMVERSION"
+    s.date = "2017-01-05"
```

Comment:
this is supposed to be the gem creation date. 
just checked the gemspec specification. I don't need to set this. the default value is good. see
[https://docs.ruby-lang.org/en/2.1.0/Gem/Specification.html#method-i-date](https://docs.ruby-lang.org/en/2.1.0/Gem/Specification.html#method-i-date) 

---
In reply to: [114165166](https://github.com/Azure/fluentd-plugin-mdsd/pull/26#discussion_r114165166) [](ancestors = 114165166)